tc/sgx/trusted_worker_manager/enclave/work_order_data_handler_wpe.cpp

```diff
@@ -0,0 +1,29 @@
+/* Copyright 2020 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include "work_order_data_handler_wpe.h"
+
+namespace tcf {
+    void WorkOrderDataHandlerWPE::InitializeDataEncryptionKey() {
```

Comment:
Add a short comment about what this is doing. Here and in the similar InitializeDataEncryptionKey() in file work_order_data_handler.cpp .
It looks like the data encryption key and IV are set to the session key and IV unless enc_data_key_str is "-", then they are set to empty.
In the latter case, does that mean there is no encryption?

Reply:
Yes Dan, if encryptedDataEncryptionKey is "-", data is sent from the requester as unencrypted text, hence there is no decryption needed for the data which is part of either inData or outData block in the work order request. 
I will mention the details in the comment.